code/API_definitions/device-reachability-status.yaml

```diff
@@ -0,0 +1,397 @@
+openapi: 3.0.3
+info:
+  title: Device Reachability Status Retrieval
+  description: |

```

Comment:
we need to add the Auth section: https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-API-access-and-user-consent.md#mandatory-template-for-infodescription-in-camara-api-specs

Reply:
Applied